scripts/resize_wrongly_scaled_cochleas.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import os
+
+import multiprocessing as mp
+from concurrent import futures
+
+import imageio.v3 as imageio
+import nifty.tools as nt
+from tqdm import tqdm
+
+from elf.wrapper.resized_volume import ResizedVolume
+from elf.io import open_file
+
+
+def main(input_path, output_folder, scale, input_key, interpolation_order):
+    if input_path.endswith(".tif"):
+        input_ = imageio.imread(input_path)
+        input_chunks = (128,) * 3
+    else:
+        input_ = open_file(input_path, "r")[input_key]
+        input_chunks = input_.chunks
+
+    abs_path = os.path.abspath(input_path)
+    basename = "".join(os.path.basename(abs_path).split(".")[:-1])
+    output_path = os.path.join(output_folder, basename + "_resized.n5")
+
+    shape = input_.shape
+    ndim = len(shape)
+
+    # Limit the number of cores for parallelization.
+    n_threads = min(16, mp.cpu_count())
+
+    shape = input_.shape
+    new_shape = tuple(
+        int(round(sh / scale)) for sh in shape
+    )
+
+    resized_volume = ResizedVolume(input_, new_shape, order=interpolation_order)
+
+    output = open_file(output_path, mode="a")
+    output_dataset = output.create_dataset(
+        input_key, shape=new_shape, dtype=input_.dtype,
+        chunks=input_chunks, compression="gzip"
+    )
+    blocking = nt.blocking([0] * ndim, new_shape, input_chunks)
+
+    def copy_chunk(block_index):
+        block = blocking.getBlock(block_index)
+        volume_index = tuple(slice(begin, end) for (begin, end) in zip(block.begin, block.end))
+        data = resized_volume[volume_index]
+        output_dataset[volume_index] = data
```

Comment:
The error was here: previously this was `output[volume_index]  = data`, which doesn't make sense.